samples/tacos/src/main/kotlin/com/slack/circuit/tacos/OrderTacosCircuit.kt

```diff
@@ -0,0 +1,356 @@
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.defaultMinSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.material.icons.filled.ArrowForward
+import androidx.compose.material3.CenterAlignedTopAppBar
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.rememberVectorPainter
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.slack.circuit.runtime.CircuitUiEvent
+import com.slack.circuit.runtime.CircuitUiState
+import com.slack.circuit.runtime.Screen
+import com.slack.circuit.runtime.presenter.Presenter
+import com.slack.circuit.tacos.model.Diet
+import com.slack.circuit.tacos.model.Ingredient
+import com.slack.circuit.tacos.step.ConfirmationOrderStep
+import com.slack.circuit.tacos.step.ConfirmationUi
+import com.slack.circuit.tacos.step.FillingsOrderStep
+import com.slack.circuit.tacos.step.FillingsUi
+import com.slack.circuit.tacos.step.OrderStep
+import com.slack.circuit.tacos.step.SummaryOrderStep
+import com.slack.circuit.tacos.step.SummaryUi
+import com.slack.circuit.tacos.step.ToppingsOrderStep
+import com.slack.circuit.tacos.step.ToppingsUi
+import com.slack.circuit.tacos.ui.theme.TacoTheme
+import java.math.BigDecimal
+import kotlinx.collections.immutable.persistentListOf
+import kotlinx.collections.immutable.persistentSetOf
+import kotlinx.collections.immutable.toImmutableSet
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+object OrderTacosScreen : Screen {
+  data class State(
+    @StringRes val headerResId: Int,
+    val orderCost: BigDecimal,
+    val stepState: OrderStep.State,
+    val isPreviousVisible: Boolean,
+    val isNextEnabled: Boolean,
+    val isNextVisible: Boolean,
+    val isFooterVisible: Boolean,
+    val eventSink: (Event) -> Unit
+  ) : CircuitUiState
+
+  sealed interface Event : CircuitUiEvent {
+    val indexModifier: Int
+      get() = 0
```

Comment:
Looks like this is always defined?

Reply:
Yup, removed. https://github.com/slackhq/circuit/pull/496/commits/b723dc52ca7f6a984b0de64d4092fd59f209db32